docs/QuickStartGuide.md

```diff
@@ -0,0 +1,62 @@
+## Quick Start Guide
+Please note that if you have installed the database with `pip` you will need to add `import building_energy_standards_data` at the beginning of the code snippets shown below. Otherwise, if you are running the following code snippets from a clone of the repository, make sure to run them from the root directory.
+### Create the Database
+```python
+import sqlite3
+from building_energy_standards_data.applications.database_maintenance import create_openstudio_standards_database_from_json
+
+conn = sqlite3.connect('openstudio_standards.db')
+create_openstudio_standards_database_from_json(conn)
+conn.close()
+```
+
+This code will generate an `openstudio_standards_database.sql` file in the same file directory. The database can be opened using a software such as [DB Browser for SQLite](https://sqlitebrowser.org/).
+### Export the Database Data
+```python
+from building_energy_standards_data.applications.database_maintenance import export_openstudio_standards_database_to_json
+conn = sqlite3.connect('openstudio_standards.db')
+export_openstudio_standards_database_to_json(conn, save_dir='./database_files/')
```

Comment:
Does this need to be `save_dir='./building_energy_standards_data/database_files/') ` since the structure of the directory has changed?

Reply:
Good catch!